docs/source/guide/GettingStarted.rst

```diff
@@ -20,6 +20,9 @@ In order to build from source, you'll need stack_:
 
     curl -sSL https://get.haskellstack.org/ | sh
 
+The centralized server also requires PostgreSQL libs. On Ubuntu, you can
```

Comment:
Ah, maybe link to here from the README then :)

Reply:
Done